### PR TITLE
Add factory creation and verifying doc

### LINKED ISSUE #9: Add the doc of creation and verifying PasskeyAccountFactory

### PR DESCRIPTION
**Pull request by oneleo**
# Add factory creation and verifying doc

## Issue linked

- Resolves #9

## Description

- Add the doc for PasskeyAccountFactory creation and verification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +26,55 @@ Clean resources for testnet:
...
+Deployed to: 0x4F0e62B7294D26b223a2cffc02BE5D072528c0De
+Transaction hash: 0xcf3c71f9b71fff6f7f45dbc3a512db667e10ab6eb272a700dfe1ab32aef0958c
+```
+
+### Verify PasskeyAccountFactory contract
+
+- Replace `<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>` with your deployed PasskeyAccountFactory address.
+
+```shell
+forge verify-contract --watch --chain "sepolia" --verifier "etherscan" --etherscan-api-key ${ETHERSCAN_API_KEY} --compiler-version "v0.8.23+commit.f704f362" --constructor-args $(cast abi-encode "constructor(address)" ${ENTRYPOINT_ADDRESS}) "<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>" "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory"
```
Comment (pilagod) [inline README.md]: nit: Couldn't `--compiler-version` just assign `v0.8.23`?

**Review by pilagod — COMMENTED** [review]
This PR seems based on branch `use-p256verifier-bytecode`, it would be better to change the merge target to `use-p256verifier-bytecode`.

File: README.md (hunk 1 above)
Comment (oneleo) [reply README.md]: After testing, the `--compiler-version` parameter using either `"0.8.23"` or `"v0.8.23"` in the `forge verify-contract` can be recognized and successfully verify the contract.
Would it be better for us to consistency in using `"0.8.23"` matching the `--use` parameter in the preceding `forge create` command?

Comment (pilagod) [reply README.md]: Definitely. We can use 0.8.23 for both places 👍

**Review by pilagod — APPROVED** [review]

**Pull request merged**